ic-os/components/guestos.bzl

```diff
@@ -34,8 +34,9 @@ component_files = {
     Label("ic/ic-https-outcalls-adapter/ic-https-outcalls-adapter.socket"): "/etc/systemd/system/ic-https-outcalls-adapter.socket",
     Label("ic/ic-https-outcalls-adapter/generate-https-outcalls-adapter-config.sh"): "/opt/ic/bin/generate-https-outcalls-adapter-config.sh",
     Label("ic/ic-replica.service"): "/etc/systemd/system/ic-replica.service",
-    Label("ic/generate-replica-config.sh"): "/opt/ic/bin/generate-replica-config.sh",
-    Label("ic/ic.json5.template"): "/opt/ic/share/ic.json5.template",
+    Label("ic/generate-ic-config/generate-ic-config.sh"): "/opt/ic/bin/generate-ic-config.sh",
```

Comment:
I renamed generate-replica-config.sh to generate-ic-config.sh because the file generates config for services other than the replica, but I don't feel strongly about this change. 

I'm happy to undo it if people think it's not worth the trouble of the change, or if people like the original name more because 'generate-ic-config' may incorrectly suggest it generates config for the whole IC-OS.